clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -473,6 +473,7 @@ def ExpansionToDefined : DiagGroup<"expansion-to-defined">;
 def FlagEnum : DiagGroup<"flag-enum">;
 def IncrementBool : DiagGroup<"increment-bool", [DeprecatedIncrementBool]>;
 def InfiniteRecursion : DiagGroup<"infinite-recursion">;
+def InvalidSpecialization : DiagGroup<"invalid-specialization">;
```

Comment:
Only 1 warning here, no reason to make a non-inline group.

Reply:
What do you mean?

Reply:
Ah, I didn't realize you could do that.